boa3_test/test_drive/testrunner/blockchain/transaction.py

```diff
@@ -66,14 +66,24 @@ def witnesses(self) -> List[Witness]:
         return self._witnesses.copy()
 
     @classmethod
-    def from_json(cls, json: Dict[str, Any]) -> TestRunnerTransaction:
+    def from_json(cls, json: Dict[str, Any], *args, **kwargs) -> TestRunnerTransaction:
```

Comment:
Same question as the `from_json `from boa3_test/test_drive/testrunner/blockchain/block.py